cargo install create-tauri-app fails

## Summary

Installation through cargo fails for me.

## Detailed info:
```sh
>rustup update
info: syncing channel updates for 'stable-x86_64-pc-windows-msvc'
info: checking for self-update

  stable-x86_64-pc-windows-msvc unchanged - rustc 1.70.0 (90c541806 2023-05-31)

info: cleaning up downloads & tmp directories
rustc 1.70.0 (90c541806 2023-05-31)

>cargo install create-tauri-app
    Updating crates.io index
  Installing create-tauri-app v3.6.1
    Updating crates.io index
   Compiling typenum v1.16.0
   Compiling winapi v0.3.9
   Compiling version_check v0.9.4
   Compiling proc-macro2 v1.0.63
   Compiling generic-array v0.14.7
   Compiling crc32fast v1.3.2
   Compiling quote v1.0.29
   Compiling unicode-ident v1.0.9
   Compiling cfg-if v1.0.0
   Compiling syn v1.0.109
   Compiling rle-decode-fast v1.0.3
   Compiling windows_x86_64_msvc v0.48.0
   Compiling proc-macro-hack v0.5.20+deprecated
   Compiling windows_x86_64_msvc v0.42.2
   Compiling libflate_lz77 v1.2.0
   Compiling winapi-util v0.1.5
   Compiling crypto-common v0.1.6
   Compiling block-buffer v0.10.4
   Compiling adler32 v1.2.0
   Compiling digest v0.10.7
   Compiling libflate v1.4.0
   Compiling same-file v1.0.6
   Compiling dirs-sys v0.3.7
   Compiling libc v0.2.147
   Compiling lazy_static v1.4.0
   Compiling cpufeatures v0.2.8
   Compiling autocfg v1.1.0
   Compiling tempfile v3.6.0
   Compiling sha2 v0.10.7
   Compiling windows-targets v0.48.1
   Compiling dirs v4.0.0
   Compiling windows-targets v0.42.2
   Compiling walkdir v2.3.3
   Compiling rust-embed-utils v7.8.1
   Compiling windows-sys v0.45.0
   Compiling shellexpand v2.1.2
   Compiling windows-sys v0.48.0
   Compiling include-flate-codegen v0.1.4
   Compiling include-flate-codegen-exports v0.1.4
   Compiling syn v2.0.22
   Compiling encode_unicode v0.3.6
   Compiling unicode-width v0.1.10
   Compiling anyhow v1.0.71
   Compiling fastrand v1.9.0
   Compiling console v0.15.7
   Compiling include-flate v0.2.0
   Compiling zeroize v1.6.0
   Compiling rust-embed-impl v6.8.1
   Compiling shell-words v1.1.0
   Compiling dialoguer v0.10.4
   Compiling pico-args v0.5.0
   Compiling rust-embed v6.8.1
   Compiling create-tauri-app v3.6.1
error: proc-macro derive panicked
  --> C:\Users\####\.cargo\registry\src\index.crates.io-6f17d22bba15001f\create-tauri-app-3.6.1\src\template.rs:12:10
   |
12 | #[derive(RustEmbed)]
   |          ^^^^^^^^^
   |
   = help: message: `folder` must be a relative path under `compression` feature.

error[E0599]: no function or associated item named `get` found for struct `FRAGMENTS` in the current scope
   --> C:\Users\####\.cargo\registry\src\index.crates.io-6f17d22bba15001f\create-tauri-app-3.6.1\src\template.rs:181:41
    |
15  | struct FRAGMENTS;
    | ---------------- function or associated item `get` not found for this struct
...
181 |         let manifest_bytes = FRAGMENTS::get(&format!("fragment-{self}/_cta_manifest_"))
    |                                         ^^^ function or associated item not found in `FRAGMENTS`
    |
    = help: items from traits can only be used if the trait is implemented and in scope
    = note: the following traits define an item `get`, perhaps you need to implement one of them:
            candidate #1: `SliceIndex`
            candidate #2: `RustEmbed`

error[E0599]: no function or associated item named `get` found for struct `FRAGMENTS` in the current scope
   --> C:\Users\####\.cargo\registry\src\index.crates.io-6f17d22bba15001f\create-tauri-app-3.6.1\src\template.rs:241:39
    |
15  | struct FRAGMENTS;
    | ---------------- function or associated item `get` not found for this struct
...
241 |             let mut data = FRAGMENTS::get(file).unwrap().data.to_vec();
    |                                       ^^^ function or associated item not found in `FRAGMENTS`
    |
    = help: items from traits can only be used if the trait is implemented and in scope
    = note: the following traits define an item `get`, perhaps you need to implement one of them:
            candidate #1: `SliceIndex`
            candidate #2: `RustEmbed`

error[E0599]: no function or associated item named `iter` found for struct `FRAGMENTS` in the current scope
   --> C:\Users\####\.cargo\registry\src\index.crates.io-6f17d22bba15001f\create-tauri-app-3.6.1\src\template.rs:276:32
    |
15  | struct FRAGMENTS;
    | ---------------- function or associated item `iter` not found for this struct
...
276 |         for file in FRAGMENTS::iter().filter(|e| {
    |                                ^^^^ function or associated item not found in `FRAGMENTS`
    |
    = help: items from traits can only be used if the trait is implemented and in scope
    = note: the following trait defines an item `iter`, perhaps you need to implement it:
            candidate #1: `RustEmbed`

error[E0277]: the size for values of type `str` cannot be known at compilation time
   --> C:\Users\####\.cargo\registry\src\index.crates.io-6f17d22bba15001f\create-tauri-app-3.6.1\src\template.rs:276:13
    |
276 |         for file in FRAGMENTS::iter().filter(|e| {
    |             ^^^^ doesn't have a size known at compile-time
    |
    = help: the trait `Sized` is not implemented for `str`
    = note: all local variables must have a statically known size
    = help: unsized locals are gated as an unstable feature

error[E0277]: the size for values of type `str` cannot be known at compilation time
   --> C:\Users\####\.cargo\registry\src\index.crates.io-6f17d22bba15001f\create-tauri-app-3.6.1\src\template.rs:276:21
    |
276 |           for file in FRAGMENTS::iter().filter(|e| {
    |  _____________________^
277 | |             path::PathBuf::from(e.to_string())
278 | |                 .components()
279 | |                 .next()
...   |
282 | |                 == "_base_"
283 | |         }) {
    | |__________^ doesn't have a size known at compile-time
    |
    = help: the trait `Sized` is not implemented for `str`
note: required by a bound in `Option`
   --> C:\Users\####\.rustup\toolchains\stable-x86_64-pc-windows-msvc\lib/rustlib/src/rust\library\core\src\option.rs:564:17
    |
564 | pub enum Option<T> {
    |                 ^ required by this bound in `Option`

error[E0277]: the size for values of type `str` cannot be known at compilation time
   --> C:\Users\####\.cargo\registry\src\index.crates.io-6f17d22bba15001f\create-tauri-app-3.6.1\src\template.rs:276:9
    |
276 | /         for file in FRAGMENTS::iter().filter(|e| {
277 | |             path::PathBuf::from(e.to_string())
278 | |                 .components()
279 | |                 .next()
...   |
284 | |             write_file(&file)?;
285 | |         }
    | |_________^ doesn't have a size known at compile-time
    |
    = help: the trait `Sized` is not implemented for `str`
note: required by a bound in `None`
   --> C:\Users\####\.rustup\toolchains\stable-x86_64-pc-windows-msvc\lib/rustlib/src/rust\library\core\src\option.rs:564:17
    |
564 | pub enum Option<T> {
    |                 ^ required by this bound in `None`

error[E0599]: no function or associated item named `iter` found for struct `FRAGMENTS` in the current scope
   --> C:\Users\####\.cargo\registry\src\index.crates.io-6f17d22bba15001f\create-tauri-app-3.6.1\src\template.rs:288:32
    |
15  | struct FRAGMENTS;
    | ---------------- function or associated item `iter` not found for this struct
...
288 |         for file in FRAGMENTS::iter().filter(|e| {
    |                                ^^^^ function or associated item not found in `FRAGMENTS`
    |
    = help: items from traits can only be used if the trait is implemented and in scope
    = note: the following trait defines an item `iter`, perhaps you need to implement it:
            candidate #1: `RustEmbed`

error[E0277]: the size for values of type `str` cannot be known at compilation time
   --> C:\Users\####\.cargo\registry\src\index.crates.io-6f17d22bba15001f\create-tauri-app-3.6.1\src\template.rs:288:13
    |
288 |         for file in FRAGMENTS::iter().filter(|e| {
    |             ^^^^ doesn't have a size known at compile-time
    |
    = help: the trait `Sized` is not implemented for `str`
    = note: all local variables must have a statically known size
    = help: unsized locals are gated as an unstable feature

error[E0277]: the size for values of type `str` cannot be known at compilation time
   --> C:\Users\####\.cargo\registry\src\index.crates.io-6f17d22bba15001f\create-tauri-app-3.6.1\src\template.rs:288:21
    |
288 |           for file in FRAGMENTS::iter().filter(|e| {
    |  _____________________^
289 | |             path::PathBuf::from(e.to_string())
290 | |                 .components()
291 | |                 .next()
...   |
294 | |                 == path::PathBuf::from(format!("fragment-{self}"))
295 | |         }) {
    | |__________^ doesn't have a size known at compile-time
    |
    = help: the trait `Sized` is not implemented for `str`
note: required by a bound in `Option`
   --> C:\Users\####\.rustup\toolchains\stable-x86_64-pc-windows-msvc\lib/rustlib/src/rust\library\core\src\option.rs:564:17
    |
564 | pub enum Option<T> {
    |                 ^ required by this bound in `Option`

error[E0277]: the size for values of type `str` cannot be known at compilation time
   --> C:\Users\####\.cargo\registry\src\index.crates.io-6f17d22bba15001f\create-tauri-app-3.6.1\src\template.rs:288:9
    |
288 | /         for file in FRAGMENTS::iter().filter(|e| {
289 | |             path::PathBuf::from(e.to_string())
290 | |                 .components()
291 | |                 .next()
...   |
296 | |             write_file(&file)?;
297 | |         }
    | |_________^ doesn't have a size known at compile-time
    |
    = help: the trait `Sized` is not implemented for `str`
note: required by a bound in `None`
   --> C:\Users\####\.rustup\toolchains\stable-x86_64-pc-windows-msvc\lib/rustlib/src/rust\library\core\src\option.rs:564:17
    |
564 | pub enum Option<T> {
    |                 ^ required by this bound in `None`

error[E0599]: no function or associated item named `get` found for struct `FRAGMENTS` in the current scope
   --> C:\Users\####\.cargo\registry\src\index.crates.io-6f17d22bba15001f\create-tauri-app-3.6.1\src\template.rs:301:35
    |
15  | struct FRAGMENTS;
    | ---------------- function or associated item `get` not found for this struct
...
301 |             let data = FRAGMENTS::get(&format!("_assets_/{src}"))
    |                                   ^^^ function or associated item not found in `FRAGMENTS`
    |
    = help: items from traits can only be used if the trait is implemented and in scope
    = note: the following traits define an item `get`, perhaps you need to implement one of them:
            candidate #1: `SliceIndex`
            candidate #2: `RustEmbed`

Some errors have detailed explanations: E0277, E0599.
For more information about an error, try `rustc --explain E0277`.
error: could not compile `create-tauri-app` (lib) due to 12 previous errors
warning: build failed, waiting for other jobs to finish...
error: failed to compile `create-tauri-app v3.6.1`, intermediate artifacts can be found at `C:\Users\####\AppData\Local\Temp\cargo-installtJfSKq`
```

--- a/.changes/rustembed-panic.md
+++ b/.changes/rustembed-panic.md
@@ -1,0 +1,6 @@
+---
+"create-tauri-app": "patch"
+"create-tauri-app-js": "patch"
+---
+
+Fix build panic with newer minor versions of `rust-embed`.

--- a/packages/cli/src/template.rs
+++ b/packages/cli/src/template.rs
@@ -10,7 +10,7 @@ use rust_embed::RustEmbed;
 use crate::{manifest::Manifest, package_manager::PackageManager};
 
 #[derive(RustEmbed)]
-#[folder = "$CARGO_MANIFEST_DIR/fragments"]
+#[folder = "fragments"]
 #[allow(clippy::upper_case_acronyms)]
 struct FRAGMENTS;
 

